app/src/main/java/kth/jjve/xfran/WorkoutsTabActivity.java

```diff
@@ -101,6 +103,31 @@ private void initNavMenu(){
         drawerLayout.addDrawerListener(toggle);
         toggle.syncState();
         navigationView.setNavigationItemSelectedListener(this);
-        navigationView.setCheckedItem(R.id.nav_home);
+        navigationView.setCheckedItem(R.id.nav_workouts);
     }
+
+    @Override
+    public void onListItemClick(int position) {
+        //List Item listener
+        //Do something on UI when the item is clicked
+        //Toast.makeText(this, mWorkoutsViewModel.getWorkouts().getValue().get(position).getType(), Toast.LENGTH_SHORT).show();
+    }
+
+    public void onPlanButtonClick(int position){
+        //TODO start activity of workout planner
+        //Intent intent = new Intent (this, HomeScreenActivity.class);
```

Comment:
Same story. Since we use version control, you can take away line 118 and line 119. The point of version control is that you can find it back in previous versions and don't have to comment stuff like this :)